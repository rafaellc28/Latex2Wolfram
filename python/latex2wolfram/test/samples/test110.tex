% Test matrix multiplication
\begin{bmatrix}
  6 & 3\\
  1 & 8
\end{bmatrix}^{-1}\cdot
\begin{bmatrix}
  6 & 3\\
  1 & 8
\end{bmatrix}
