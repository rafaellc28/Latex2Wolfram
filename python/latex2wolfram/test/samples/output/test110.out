{{6, 3}, {1, 8}} ^ (-1) . {{6, 3}, {1, 8}}
